src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -42,8 +43,8 @@ public EventCard(Event event, int displayedIndex) {
         date.setText(event.getDate().toString());
         time.setText(event.getTime().toString());
         event.getParticipants().stream()
-                .sorted(Comparator.comparing(participant -> participant.getName().fullName))
-                .forEach(participant -> participants.getChildren().add(new Label(participant.getName().fullName)));
+                .sorted(Comparator.comparing(Person::getFullName))
+                .forEach(participant -> participants.getChildren().add(new Label(participant.getFullName())));
```

Comment:
Is it possible to use `Particpant::getFullName` on line 46 instead? Athough it doesn't matter now (cuz they are essentially the same method), but I think it will better facilitate the removal of Person class in the future. (Im not sure if this is possible 😅 )

Reply:
Good point, not sure why Person class was used here, will get this corrected